src/_gettsim/social_insurance_contributions/arbeitsl_v.py

```diff
@@ -190,34 +191,39 @@ def _arbeitsl_v_beitr_midijob_sum_arbeitn_arbeitg_m(
     out = (
         midijob_bemessungsentgelt_m
         * 2
-        * soz_vers_beitr_params["beitr_satz"]["arbeitsl_v"]
+        * sozialv_beitr_params["beitr_satz"]["arbeitsl_v"]
     )
     return out
 
 
-def _arbeitsl_v_beitr_midijob_arbeitg_m_bis_09_2022(
+@dates_active(
+    end="2022-09-30",
+    change_name="_arbeitsl_v_beitr_midijob_arbeitg_m",
+)
+def _arbeitsl_v_beitr_midijob_arbeitg_m_anteil_bruttolohn(
     bruttolohn_m: float,
-    soz_vers_beitr_params: dict,
+    sozialv_beitr_params: dict,
 ) -> float:
     """Calculating the employer unemployment insurance contribution until September
     2022.
 
     Parameters
     ----------
-    soz_vers_beitr_params
-        See params documentation :ref:`soz_vers_beitr_params <soz_vers_beitr_params>`.
+    sozialv_beitr_params
+        See params documentation :ref:`sozialv_beitr_params <sozialv_beitr_params>`.
     bruttolohn_m
         See basic input variable :ref:`bruttolohn_m <bruttolohn_m>`.
 
     Returns
     -------
 
     """
-    out = bruttolohn_m * soz_vers_beitr_params["beitr_satz"]["arbeitsl_v"]
+    out = bruttolohn_m * sozialv_beitr_params["beitr_satz"]["arbeitsl_v"]
     return out
 
 
-def _arbeitsl_v_beitr_midijob_arbeitg_m_ab_10_2022(
+@dates_active(start="2022-10-01", change_name="_arbeitsl_v_beitr_midijob_arbeitg_m")
+def _arbeitsl_v_beitr_midijob_arbeitg_m_residuum(
```

Comment:
Good point! Though I think the best way will be to just incorporate the information from `dates_active` in the docstring instead of repeating. 

@lars-reimann @mj023, we should discuss this at some point.